app/models/form_submission_attempt.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class FormSubmissionAttempt < ApplicationRecord
+  include AASM
+
+  belongs_to :form_submission
+  has_one :saved_claim, through: :form_submission
```

Comment:
If the associations for these on the `form_submission` are optional, does that screw up these `through` associations if any of them are nil?

Reply:
I may be mistaken about this so please check me but this is taken from Rails docs:
<img width="686" alt="Screenshot 2023-12-11 at 1 52 07 PM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/10481391/bc290c13-7d61-4e69-a3cf-6c4b8c3c7a3d">
https://guides.rubyonrails.org/association_basics.html#the-has-many-through-association
My reading of that indicates that the nil associations are automatically handled?

Reply:
You could test by creating a form submission with one of the nil associations and confirm `form_submission_attempt.saved_claim` or whatever returns `nil` as expected instead of an error (but I agree with you it probably works fine)